cmd/server/main.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/golang/glog"
+	"github.com/kubernetes-sigs/aws_encryption-provider/cloud"
+	"github.com/kubernetes-sigs/aws_encryption-provider/connection"
+	"github.com/kubernetes-sigs/aws_encryption-provider/plugin"
+	"github.com/kubernetes-sigs/aws_encryption-provider/server"
+)
+
+func main() {
+	var (
+		healthzPath = flag.String("healthz-path", "/healthz", "healthcheck path")
+		healthzPort = flag.String("health-port", ":8080", "healthcheck port")
+		addr        = flag.String("listen", "/tmp/awsencryptionprovider.sock", "GRPC listen address")
```

Comment:
Debating whether we should match this to the path used by the google provider.  Not a blocker though.